api/api/models/media.py

```diff
@@ -65,19 +70,37 @@ class AbstractMedia(
         "Source and provider can be different. Eg: the Google Open "
         "Images dataset is source=openimages, but provider=flickr.",
     )
-    last_synced_with_source = models.DateTimeField(blank=True, null=True, db_index=True)
-    removed_from_source = models.BooleanField(default=False)
-
-    view_count = models.IntegerField(
+    last_synced_with_source = models.DateTimeField(
         blank=True,
         null=True,
-        default=0,
+        db_index=True,
+        help_text="The date the media was last updated from the upstream source.",
+    )
+    removed_from_source = models.BooleanField(
+        default=False,
+        help_text="Whether or not the media has been removed from the upstream source.",
+    )
+
+    view_count = models.IntegerField(
+        blank=True, null=True, default=0, help_text="Vestigial field, purpose unknown."
     )
 
     tags = models.JSONField(
         blank=True,
         null=True,
-        help_text="Tags with detailed metadata, such as accuracy.",
+        help_text=d(
+            """
```

Comment:
Consistency nit.

```suggestion
        help_text=d("""
```

This text is not necessary, in my opinion, and duplicates the documentation of the tag schema. Note the "tags" schema embedded below.

![20240430_09h07m43s_grim](https://github.com/WordPress/openverse/assets/24264157/471674fb-3fe5-41f8-bf4b-10445361492b)

If we actually want to improve this field's documentation, we should remove the suggestion of particular individual properties from this help text, and instead augment whatever improvements we wish to represent on the Tag schema.